crates/shared/src/arguments.rs

```diff
@@ -509,8 +510,10 @@ pub fn duration_from_seconds(s: &str) -> Result<Duration, ParseFloatError> {
     Ok(Duration::from_secs_f64(s.parse()?))
 }
 
-pub fn wei_from_base_unit(s: &str) -> anyhow::Result<U256> {
-    Ok(U256::from_dec_str(s)? * U256::exp10(18))
+pub fn wei_from_ether(s: &str) -> anyhow::Result<U256> {
+    let in_ether = s.parse::<BigDecimal>()?;
+    let base = BigDecimal::new(1.into(), -18);
+    number_conversions::big_decimal_to_u256(&(in_ether * base)).context("invalid Ether value")
```

Comment:
Why? Fractional support?

Reply:
Exactly. We probably want this to be around 0.01 Ether or something.

Reply:
Fixed the default I use in the command line arguments (was set to 20 Eth 🙈).